jekyll/_cci2/remote-docker-images-support-policy.adoc

```diff
@@ -35,22 +35,18 @@ We support various tags for the remote Docker environment to allow you to choose
 
 For the latest major version of Docker:
```

Comment:
Overview needs to be fixed to match new policy with some minor changes:

`This document outlines the [CircleCI remote Docker image](https://circleci.com/docs/building-docker-images/) release, update, and deprecation policy. This policy applies to all CircleCI remote Docker images built for the remote Docker feature (setup_remote_docker).`